testsuites/syncgateway/functional/tests/conftest.py

```diff
@@ -298,3 +298,6 @@ def params_from_base_test_setup(request, params_from_base_suite_setup):
         logging_helper.fetch_and_analyze_logs(cluster_config=cluster_config, test_name=test_name)
 
     assert len(errors) == 0
+
+    # Stop all sync_gateway and sg_accels as test finished
+    c.stop_sg_and_accel()
```

Comment:
We can call methods from SyncGateway.py as detailed above here depending on the mode.

Reply:
same as above